manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -33,6 +33,8 @@ gateways:
         memory: 1024Mi
     cpu:
       targetAverageUtilization: 80
+    memory:
+      targetAverageUtilization: 80
```

Comment:
This makes it _required_ to set a memory target. This should be opt in.

Reply:
Okay, I'll modify it to apply the option selectively.